### PR TITLE
ORM 2 objectives

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[EPIC - Java Lesson 17 🎆](https://trello.com/c/TQv0SGto/370-epic-java-lesson-17-%F0%9F%8E%86)

File: unit2-java/lesson16.rst [hunk 1]
```diff
@@ -5,24 +5,19 @@ Goals
...
 
 - Understand how object-relational mapping allows object-oriented web applications to store data in relational databases
-- Implement ORM with one-to-one and one-to-many relationships in Spring Boot
+- Implement ORM for individual, non-related entities in Spring Boot
 
 Objectives
 ----------
 
 - Explain what object-relational mapping refers to
 - Describe what a data layer is and how it is used in ORM
```
Comment (jimflores5) [inline unit2-java/lesson16.rst]: Comma addition: "...layer is, and how..."

File: unit2-java/lesson16.rst [hunk 2]
```diff
@@ -5,24 +5,19 @@ Goals
 -----
 
 - Understand how object-relational mapping allows object-oriented web applications to store data in relational databases
-- Implement ORM with one-to-one and one-to-many relationships in Spring Boot
+- Implement ORM for individual, non-related entities in Spring Boot
 
 Objectives
 ----------
 
 - Explain what object-relational mapping refers to
```
Comment (jimflores5) [inline unit2-java/lesson16.rst]: I'm not sure of the grammar acceptability of ending a sentence with "to".  Perhaps, "...mapping means"?

File: unit2-java/lesson16.rst [hunk 3]
```diff
@@ -5,24 +5,19 @@ Goals
...
 - Describe what a data layer is and how it is used in ORM
-- Describe the one-to-many relationship
 - Use the following annotations and describe what they accomplish:
```
Comment (jimflores5) [inline unit2-java/lesson16.rst:15]: This sentence is rendered as bold and italic. I think you need to add an empty line between it and the sub-list.

File: unit2-java/lesson17.rst [hunk 4]
```diff
@@ -4,14 +4,28 @@ Unit 2 (Java) - Lesson 17 Learning Objectives
...
+- Understand how to model relationships in a relational database
+
 Objectives
 ----------
 
-- Describe the many-to-many relationship and give an example of when this may be a good way to represent data in your database
+One-to-many Relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Describe the one-to-many relationship
```
Comment (jimflores5) [inline unit2-java/lesson17.rst]: This was done in the SQL lessons.

File: unit2-java/lesson17.rst [hunk 5]
```diff
@@ -4,14 +4,28 @@ Unit 2 (Java) - Lesson 17 Learning Objectives
...
 -----
 
+- Build persistent model classes that have one-to-many and many-to-many relationships
+- Understand how to model relationships in a relational database
+
 Objectives
 ----------
 
-- Describe the many-to-many relationship and give an example of when this may be a good way to represent data in your database
+One-to-many Relationships
```
Comment (jimflores5) [inline unit2-java/lesson17.rst]: Title Case?  "One-to-Many Relationships"

File: unit2-java/lesson17.rst [hunk 6]
```diff
@@ -4,14 +4,28 @@ Unit 2 (Java) - Lesson 17 Learning Objectives
...
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Describe the one-to-many relationship
+- Use the following annotations and describe what they accomplish:
```
Comment (jimflores5) [inline unit2-java/lesson17.rst:17]: This line is rendered bold and italic.  Add an empty line between it and the sub-list.

File: unit2-java/lesson17.rst [hunk 7]
```diff
@@ -4,14 +4,28 @@ Unit 2 (Java) - Lesson 17 Learning Objectives
...
+   - ``@OneToMany``
+   - ``@JoinColumn``
+   - ``@ManyToOne``
+
+Many-to-many Relationships
```
Comment (jimflores5) [inline unit2-java/lesson17.rst]: "Many-to-Many"?

**Review by jimflores5 — CHANGES_REQUESTED** [review]

**Pull request merged**